packages/tendermint-rpc/types/adaptors/v0-33/hasher.spec.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Is this here because of the additional nesting?

Reply:
I think so, yes. Unfortunately the `/**/` path component does not work well in shx. Any ideas how to best fix this?

Reply:
I mean, manually add additional levels? It's not pretty but we're not likely to get much deeper than this any time soon.

Reply:
If we had all our tests in a `__tests__` directory this wouldn't be an issue.